static/index.js

```diff
@@ -0,0 +1,150 @@
+function renderPage() {
+   let reviews = getCurrentReviews();
+   let reviewWrapper = document.getElementById('reviews');
+   reviewWrapper.innerHTML = "";
+
+   reviews.forEach((review) => {
+      reviewWrapper.appendChild(createReviewDiv(review, reviews));
+   });
+
+   let averageRating = document.getElementById('average-rating');
+   averageRating.textContent = `${getAverage().toFixed(1)} / 5`;
+   show.appendChild(averageRating);
+
+   setUpStars();
+}
+
+function createReviewDiv(review, reviews) { 
+   // saljem reviews kao argument da se reference poklapaju za filter
+
+   let reviewDiv = document.createElement('div');
+   reviewDiv.classList = ['review round'];
+   let reviewText = document.createElement('p');
+   reviewText.textContent = review.text;
+   let reviewRating = document.createElement('p');
+   reviewRating.textContent = `${review.rating} / 5`;
+
+
+   // dodavanje zvjezdica na reviewove
+   let reviewStars = document.createElement('div');
+   for (let i = 1; i <= 5; i++) {
+      let star = document.createElement('img');
+      star.src = (i <= review.rating) ? 'images/star-yellow.png' : 'images/star-white.png';
+      star.classList = ['stars'];   // da se primjeni stil za klasu stars
+
+      reviewStars.appendChild(star);
+   }
+
+   let removeButton = document.createElement('input');

```

Comment:
Postoji neki razlog zašto ovo nisu button elementi?
Ovo je okej, ali mislim da je jednostavnije koristiti button tag kao takav (ili ja možda nešto propuštam).
Prođemo na predaji :)